webapp/src/App.js

```diff
@@ -67,47 +72,90 @@ class App extends Component {
             >
               <span class="navbar-toggler-icon" />
             </button>
-            <div class="collapse navbar-collapse" id="navbarNav">
+            <div
+              class={
+                "collapse navbar-collapse" +
+                (this.state.collapsed ? " collapsed" : "")
+              }
+              id="navbarNav"
+            >
               <ul class="navbar-nav mr-auto">
-                <li class="nav-item">
+                <li class={"nav-item"}>
                   <NavLink
                     exact
                     to="/"
                     activeClassName="nav-link active"
                     className="nav-link"
+                    onClick={this.toggleMenu}
                   >
                     Home
                   </NavLink>
                 </li>
-                {this.state.user && <li class="nav-item">
-                  <NavLink
-                    to="/applicationForm"
-                    activeClassName="nav-link active"
-                    className="nav-link"
-                  >
-                    Apply
-                  </NavLink>
-                </li>}
+                {this.state.user && (
+                  <li class="nav-item">
+                    <NavLink
+                      to="/applicationForm"
+                      activeClassName="nav-link active"
+                      className="nav-link"
+                      onClick={this.toggleMenu}
+                    >
+                      Apply
+                    </NavLink>
+                  </li>
+                )}
               </ul>
-              <UserDropdown logout={this.refreshUser} user={this.state.user} />
+              <UserDropdown
+                logout={this.refreshUser}
+                user={this.state.user}
+                onClick={this.toggleMenu}
```

Comment:
Haven't tested this, but how will this affect the dropdown when the user is logged in?

Reply:
Good point. Will test this tonight.

Reply:
Yep, works as expected.